books/free-programming-books-langs.md

```diff
@@ -400,12 +400,14 @@ Books on general-purpose programming that don't focus on a specific language are
 * [Google Shell Style Guide](https://google.github.io/styleguide/shell.xml) - Paul Armstrong (HTML)
 * [Introduction to Bash Scripting](https://github.com/bobbyiliev/introduction-to-bash-scripting) - Bobby Iliev (Markdown, PDF)
 * [Introduction to the Command Line](https://launchschool.com/books/command_line) - Launch School (HTML)
+* [Learning the bash Shell, 3rd Edition - Unix Shell Programming](https://doc.lagout.org/operating%20system%20/linux/Learning%20the%20bash%20Shell%20-%20Unix%20Shell%20Programming.pdf) - Cameron Newham (PDF)
```

Comment:
This is not an authorized distribution. Refer to docs/CONTRIBUTING.md for guidelines.